### PR TITLE
Support Adobe Magento CSV language packages 

### PR DESCRIPTION
**Pull request by aurambaj**
The CSV has 2 columns: source, target. The filename is made of the locale only with "_" and is inside the "i18n" directory

See reference documentation: https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/translations/xlate.html#m2devgde-xlate-languagepack

### CODE REVIEW

File: common/src/main/resources/com/box/l10n/mojito/okapi/filters/okf_table_csv_mojito_adobe_magento.fprm [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#v1
+fieldDelimiter=,
+textQualifier="
+sourceIdColumns=1
+sourceColumns=1
+targetColumns=2
```
Comment (hylstonnb) [inline common/src/main/resources/com/box/l10n/mojito/okapi/filters/okf_table_csv_mojito_adobe_magento.fprm:6]: Could you let me know why targetColumns is 2 and not 1?

File: webapp/package-lock.json [hunk 2]
```diff
@@ -1044,9 +1044,9 @@
       },
       "dependencies": {
         "minimist": {
-          "version": "0.2.2",
-          "resolved": "https://registry.npmjs.org/minimist/-/minimist-0.2.2.tgz",
-          "integrity": "sha512-g92kDfAOAszDRtHNagjZPPI/9lfOFaRBL/Ud6Z0RKZua/x+49awTydZLh5Gkhb80Xy5hmcvZNLGzscW5n5yd0g=="
+          "version": "0.2.4",
```
Comment (hylstonnb) [inline webapp/package-lock.json:1047]: is `package-lock.json` file commit intentional?

**Review by hylstonnb — COMMENTED** [review]
In general LGTM, just have a couple comments

File: webapp/package-lock.json (hunk 2 above)
Comment (aurambaj) [reply webapp/package-lock.json:1047]: nope, thanks for flagging.

File: common/src/main/resources/com/box/l10n/mojito/okapi/filters/okf_table_csv_mojito_adobe_magento.fprm (hunk 1 above)
Comment (aurambaj) [reply common/src/main/resources/com/box/l10n/mojito/okapi/filters/okf_table_csv_mojito_adobe_magento.fprm:6]: it is the index of the column where the target string will be. If you look at the `fr_FR.csv`, you can see the first column is the source (english) and the second column is the translation.

Comment (hylstonnb) [reply common/src/main/resources/com/box/l10n/mojito/okapi/filters/okf_table_csv_mojito_adobe_magento.fprm:6]: ok that is the column index, makes sense, thanks

**Pull request merged**